myst_nbs/case_studies/stochastic_volatility.myst.md

```diff
@@ -182,9 +189,25 @@ axes[1].set_title("Posterior volatility");
 
 ## References
 
-1. Hoffman & Gelman. (2011). [The No-U-Turn Sampler: Adaptively Setting Path Lengths in Hamiltonian Monte Carlo](http://arxiv.org/abs/1111.4246).
+:::{bibliography}
+:filter: docname in docnames
+:::
 
 ```{code-cell} ipython3
```

Comment:
There should be a markdown cell with the "watermark" title before this code cell: https://docs.pymc.io/en/latest/contributing/jupyter_style.html#watermark